docs/source/secrets.rst

```diff
@@ -1,33 +0,0 @@
-Secrets
```

Comment:
This page described the old way of storing secret info, and should have been removed when the outputs refactor happened but was overlooked.

We no longer require the user to use `aws kms` from the command line and handle all the overhead for them.